front/production_checks/checks/check_notion_active_workflows.ts

```diff
@@ -31,26 +25,31 @@ async function listAllNotionConnectors() {
   return notionConnectors;
 }
 
-async function isTemporalWorkflowRunning(
+async function areTemporalWorkflowsRunning(
   client: Client,
   notionConnector: NotionConnector
 ) {
   try {
-    const handle: WorkflowHandle = client.workflow.getHandle(
-      getWorkflowId(notionConnector)
+    const incrementalSyncHandle: WorkflowHandle = client.workflow.getHandle(
+      getNotionWorkflowId(notionConnector, "never")
+    );
+    const garbageCollectorHandle: WorkflowHandle = client.workflow.getHandle(
+      getNotionWorkflowId(notionConnector, "always")
     );
 
-    const description = await handle.describe();
-    const { status } = description;
+    const descriptions = await Promise.all([
```

Comment:
By combining the two checks into a single `Promise.all()`, we somewhat lose track of which specific workflow isn't running...

Reply:
Agree, but it's IMO enough to know that one of them isn't running (we filter by connectorId in temporal anyway + the command to restart will restart both workflows anyway)